ui/ducks/send/send.js

```diff
@@ -2336,10 +2360,17 @@ export function signTransaction() {
           : TransactionType.simpleSend;
 
       if (draftTransaction.asset.type !== AssetType.native) {
-        transactionType =
-          draftTransaction.asset.type === AssetType.NFT
-            ? TransactionType.tokenMethodTransferFrom
-            : TransactionType.tokenMethodTransfer;
+        if (draftTransaction.asset.type === AssetType.NFT) {
+          if (
+            draftTransaction.asset.details.standard === TokenStandard.ERC721
+          ) {
+            transactionType = TransactionType.tokenMethodTransferFrom;
+          } else {
+            transactionType = TransactionType.tokenMethodSafeTransferFrom;
+          }
+        } else {
+          transactionType = TransactionType.tokenMethodTransfer;
+        }
```

Comment:
Alternative formatting option that I find more readable, but I know not everyone likes ternary.  Up to you.

```suggestion
        transactionType =
          draftTransaction.asset.type === AssetType.NFT
            ? draftTransaction.asset.details.standard === TokenStandard.ERC721
              ? TransactionType.tokenMethodTransferFrom
              : TransactionType.tokenMethodSafeTransferFrom
            : TransactionType.tokenMethodTransfer;
```